content/en/guides/deployment/deployment-koyeb.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Deploy Nuxt on Koyeb Serverless Platform
+description: Deploy Nuxt on Koyeb Serverless Platform with Docker
+menu: Koyeb
+target: Server
+category: deployment
+position: 110.1
+---
+
+[Koyeb](https://www.koyeb.com) is a developer-friendly serverless platform to deploy apps globally. The platform lets you seamlessly run Docker containers, web apps, and APIs with git-based deployment, native autoscaling, a global edge network, and built-in service mesh and discovery.
+
+In this guide, we showcase how to dockerize and deploy a Nuxt application on the Koyeb platform. Koyeb allows you to deploy Docker containers from the registry of your choice. In this guide, we will push our Nuxt dockerized application on the Docker Hub and deploy it on Koyeb.
+
+## Requirements
+
+To successfully follow and complete this guide, you need:
+
+1. To have a Nuxt project to deploy. You can use the [create-nuxt-app](https://github.com/nuxt/create-nuxt-app) to create a Nuxt project and get started
+2. A [Koyeb account](https://app.koyeb.com) to deploy and run the dockerized Nuxt application
+3. Have [Docker Hub] account to push the Docker image and deploy it on Koyeb
+
+## Getting started
+
+In your Nuxt application directory run the following command to install dependencies:
+
+```bash
+# Using yarn
+yarn
+
+# Using npm
+npm install
+```
+
+Once the dependencies are installed, launch your application and ensure everything is working fine:
+
+```bash
+# Using yarn
+
+yarn dev
+
+# Using npm
+
+npm run dev
+```
+
+## Dockerize your application
+
+To Dockerize your Nuxt application, you need to create a `Dockerfile` in your project directory containing the content below:
+
+```dockerfile
+FROM node:lts
+
+WORKDIR /app
+
+COPY . ./
```

Comment:
Single step build will include all dev/build dependencies to final image. Can we update it with a Dockerfile like [this](https://github.com/nuxt/nuxt.js/blob/dev/examples/docker-build/Dockerfile)?

Reply:
Will update to use multi-stage build.